sysprep/activate_instance.ps1

```diff
@@ -273,29 +273,28 @@ function Verify-ActivationStatus {
       $slmgr_status = & cscript //E:VBScript //nologo $env:windir\system32\slmgr.vbs /dli
     }
     catch {
-      return $active
+      $active = $false
     }
     $status = $slmgr_status | Select-String -Pattern '^License Status:'
     # The initial space is to ensure "Unlicensed" does not match.
     if ($status -match ' Licensed') {
       $active = $true
     }
-    return $active
   }
   # All server versions newer than 2008
   else {
     try {
       $activation_status = (Get-ItemProperty -Path "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\SoftwareProtectionPlatform\Activation" -Name ProductActivationResult).ProductActivationResult
     }
     catch {
-      return $active
+      $active = $false
```

Comment:
The variable is already set to false and only a positive result would set it true.

I'd suggest the catch simply log the issue.

Reply:
I changed it to write to host.